v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -29,17 +29,35 @@
 
   void setTargetNodeAddresses(String targetNodeAddresses);
 
-  @Description("The index toward which the requests will be issued, ex: my-index")
+  @Description("The document type toward which the requests will be issued, ex: my-document-type")
```

Comment:
Can we remove this all together? This is not an option any more because the document type will be `_doc` and implicitly added by the code. This is the guidance for all new ES versions